resources.md

```diff
@@ -42,6 +42,7 @@ programming languages, and programs using the related platform:
 * [trino-go-client]({{site.github_org_url}}/trino-go-client) for Go
 * [presto-client-node](https://github.com/tagomoris/presto-client-node) for Node.js
 * [lento](https://github.com/vweevers/lento) for Node.js
+* [dbt client](https://github.com/starburstdata/dbt-trino) for Python
```

Comment:
This is more than a client library. It's an adapter for dbt, so it should probably go in a different section.

Reply:
well .. we don't really have such a section ... and some of the others clients are also close to being adapters ... its a blurry line .. should we change the title of this section maybe? Or do you think it warrants another section?

Reply:
but yes ... it should be renamed to `dbt adapter` for the link text .. and maybe we should drop "for Python"

Reply:
The closest fit would be https://trino.io/resources.html#ui ....

Reply:
I changed the text to "dbt adapter" and removed the "for Python"

Would it make sense to change this section title to "Client libraries and adapters" or perhaps add a subsection under "Client libraries" for adapters?